mysql/README.md

```diff
@@ -1,59 +1,131 @@
 # Overview
 
-Connect MySQL to Datadog in order to:
+The Datadog agent's MySQL check sends many database metrics to Datadog, including metrics for:
 
-* Visualize your database performance
-* Correlate the performance of MySQL with the rest of your applications
+* Query throughput
+* Query performance (average query run time, slow queries, etc)
+* Connections (currently open connections, aborted connections, errors, etc)
+* InnoDB (buffer pool metrics, etc)
 
-# Installation
+And [many more](https://github.com/DataDog/integrations-core/blob/master/mysql/metadata.csv). You can also invent your own metrics using custom queries.
```

Comment:
Yeah, I'm not sure how we'll want to deal with this yet. Maybe cross that bridge when we come to it?

Readers of the integrations-core READMEs need to know to look at metadata.csv for their list of metrics, but maybe linking it mid-sentence like this will be too vexing for the future Docs to detect when scraping. One thought is to write whatever prose we like in the main body of Overview - i.e. 'and many more' (no link) - and then have a subheading (an H3, say) within Overview which references metadata.csv and any other reference docs. It may be easier for the New Docs to ignore the entire H3 than to ferret out links to reference docs in the main Overview body.

Alternatively we could have another top-level heading only for integrations-core where we point users to the reference docs, but I like pointing out to the reader up front that the main data listed in Overview are not the *only* metrics/events/service checks.

TL;DR maybe cross the bridge when we come to it.